main.tf

```diff
@@ -0,0 +1,101 @@
+# Merge the provided ENV vars with EFS_HOST (created EFS DNS hostname)
+resource "null_resource" "env_vars" {
+  triggers = {
+    value = "${
+    merge(
+      map(
+        "EFS_HOST", "${module.efs.host}"
+      ), var.env_vars
+    )
+  }"
+  }
+
+  lifecycle {
+    create_before_destroy = true
+  }
+
+  depends_on = ["module.efs"]
+}
+
+# Elastic Beanstalk Application
+module "elastic_beanstalk_application" {
+  source      = "git::https://github.com/cloudposse/terraform-aws-elastic-beanstalk-application.git?ref=tags/0.1.2"
+  namespace   = "${var.namespace}"
+  name        = "${var.name}"
+  stage       = "${var.stage}"
+  description = "${var.description}"
+}
+
+# Elastic Beanstalk Environment
+module "elastic_beanstalk_environment" {
+  source                  = "git::https://github.com/cloudposse/terraform-aws-elastic-beanstalk-environment.git?ref=tags/0.2.1"
+  attributes              = ["eb"]
+  namespace               = "${var.namespace}"
+  name                    = "${var.name}"
+  stage                   = "${var.stage}"
+  zone_id                 = "${var.zone_id}"
+  app                     = "${module.elastic_beanstalk_application.app_name}"
+  instance_type           = "${var.master_instance_type}"
+  autoscale_min           = 1
```

Comment:
Maybe add a comment as to why these are hardcoded and why they should not be changed.

Reply:
Commented